test/integration/targets/timezone/tasks/main.yml

```diff
@@ -1,15 +1,20 @@
-- name: set timezone to Etc/UTC
-  timezone:
-    name: Etc/UTC
+- block:
+    - name: set timezone to Etc/UTC
+      timezone:
+        name: Etc/UTC
 
-- name: set timezone to Australia/Brisbane
-  timezone:
-    name: Australia/Brisbane
-  register: timezone_set
+    - name: set timezone to Australia/Brisbane
+      timezone:
+        name: Australia/Brisbane
+      register: timezone_set
 
-- name: ensure timezone changed
-  assert:
-    that:
-      - timezone_set.changed
-      - timezone_set.diff.after.name == 'Australia/Brisbane'
-      - timezone_set.diff.before.name == 'Etc/UTC'
+    - name: ensure timezone changed
+      assert:
+        that:
+          - timezone_set is changed
+          - timezone_set.diff.after.name == 'Australia/Brisbane'
+          - timezone_set.diff.before.name == 'Etc/UTC'
+  always:
+    - name: Return timezone to UTC
+      timezone:
+        name: Etc/UTC
```

Comment:
Restore the timezone to the original value registered at the start of the test.

Reply:
Done